### DISCUSSION TITLE
SQLitePCLRaw.nativelibrary.dll library missing after successful build

### DISCUSSION THREAD

**Discussion by rahulryadav** [ericsink/SQLitePCL.raw#488] [Q&A]
I am currently working to Publish NuGet Package with signed version of assemblies from this project. I have cloned the repository and able to successfully build the projects locally after excluding the following projects which are not part of my requirement:

//"lib.e_sqlite3.android"
//"lib.e_sqlite3.ios"
//"lib.e_sqlite3.tvos"
//"lib.e_sqlcipher.android"
//"lib.e_sqlcipher.ios"

But after a successful build I am not able to locate the SQLitePCLRaw.nativelibrary.dll.

I checked following things for more information:
SQLitePCLRaw.nativelibrary.csproj file does not exist but it is being referred in the tests.csproj project file. Also searched the recent build logs for this repository in Actions menu but did not find anything related to this assembly.

Can someone help me to understand what is the source of SQLitePCLRaw.nativelibrary.dll and how it is created and packaged with NuGet? Also let me know if it has some dependencies with projects which I have excluded from build process.

**Comment by ericsink** [comment]
Is this related to #490 ?

**Reply by rahulryadav** [reply]
I found the SQLitePCLRaw.nativelibrary project and directory in the https://github.com/ericsink/SQLitePCL.raw/releases/tag/v2.1.0 sourcecode.zip file. It was missing in the master branch. After taking this sourcecode.zip above mentioned issue is resolved.

Thanks
